core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -78,50 +84,36 @@
    */
   public static final class Scopes {
     /**
-     * @return A set of scopes including all masters, workers and clients
+     * A set of scopes including all masters, workers and clients.
```

Comment:
Remove javadoc - it doesn't add anything new and doubles the number of lines in the class